spring-cloud-dataflow-dependencies/pom.xml

```diff
@@ -20,7 +20,7 @@
 		<spring-batch-admin-dependencies.version>1.3.1.RELEASE</spring-batch-admin-dependencies.version>
 		<spring-cloud-dataflow-ui.version>1.0.0.RELEASE</spring-cloud-dataflow-ui.version>
 		<spring-shell.version>1.2.0.RELEASE</spring-shell.version>
-		<spring-cloud-deployer.version>1.0.1.RELEASE</spring-cloud-deployer.version>
+		<spring-cloud-deployer.version>1.0.2.BUILD-SNAPSHOT</spring-cloud-deployer.version>
```

Comment:
Not sure if the recent [changes] (https://github.com/spring-cloud/spring-cloud-deployer/commit/84bbf2a850e1ee7e21a9265718ecea7d4f28dc1f) on `spring-cloud-deployer` is critical enough to make it to GA. But, creating this PR in case if we decide so.